serverless/src/main.rs

```diff
@@ -160,6 +160,7 @@ async fn start_dogstatsd(
                 Arc::clone(&metrics_aggr),
                 build_fqdn_metrics(dd_site),
                 https_proxy,
+                Duration::from_secs(5),
```

Comment:
nitpick: I typically like to put these sorts of values in consts so it's clearer what their purpose is.

Reply:
On it